public/lessons/javascript_intro/common_methods.md

```diff
@@ -42,5 +42,63 @@ returns a capitalized version of the string it's called on
 returns an array of strings split at the specified character
 
 ```javascript
-'blueberry'.split('e') //returns ['blu', 'b', 'rry']
+'blueberry'.split('e') // => ['blu', 'b', 'rry']
 ```
+# Methods on Numbers
+
+Numbers also have some methods that can be called on them.  There is also a globally available `Math` class available in JavaScript which has a number of usefull methods attached to it
+
+# .toString()
+
+takes a number object and turns it into a string
+
+```javascript
+let x = 10
+x.toString() // => '10'
+
+(42).toString() // => '42'
```

Comment:
You may want to make a note as to why Numbers need to be wrapped in parenthesis when calling methods on the literal value. Students sometimes get confused as to why `42.toString()` raises an error instead of returning a value.